core/src/main/java/jenkins/security/RekeySecretAdminMonitor.java

```diff
@@ -144,7 +144,7 @@ protected void fix(TaskListener listener) throws Exception {
             int count = rewriter.rewriteRecursive(Jenkins.get().getRootDir(), listener);
             log.printf("Completed re-keying %d files on %s\n",count,new Date());
             new RekeySecretAdminMonitor().done.on();
-            LOGGER.info("Secret re-keying completed");
+            LOGGER.finest("Secret re-keying completed");
```

Comment:
Basically dead code so 🤷‍♂ 

But I remember this taking _forever_, so a visible-by-default log message could be considered helpful.